game/src/constants.js

```diff
@@ -7,5 +7,6 @@ const greetings = [
     '1000% more Cube!',
     'A Whole New Dimension!',
     'Are These Voxels?',
+    'a = -9.8m/s^2'
```

Comment:
There needs to be a comma at the end of the line as this is a list. Only the last entry doesn't need a comma